plopfile.js

```diff
@@ -0,0 +1,82 @@
+const promptRecursive = require('inquirer-recursive');
+const { inputRequired } = require('./config/plop/utils/utils');
+
+module.exports = (plop) => {
+  plop.setPrompt('recursive', promptRecursive);
+
+  plop.setGenerator('Store generator', {
```

Comment:
`plop.setGenerator('add:store', {` - so it will be used in package.json script to run exact store creation with `npm run add:store`